sdk/keyvault/keyvault-admin/rollup.base.config.js

```diff
@@ -66,7 +66,7 @@ export function nodeConfig(test = false) {
     // different output file
     baseConfig.output.file = "dist-test/index.node.js";
 
-    baseConfig.external.push("assert", "fs", "path");
+    baseConfig.external.push("assert", "fs", "path", "chai");
```

Comment:
we mark all dev deps as external in TA [here](https://github.com/deyaaeldeen/azure-sdk-for-js/blob/823ca076fd13842ab096fed514351fe1344b4d06/sdk/textanalytics/ai-text-analytics/rollup.base.config.js#L46) cc @willmtemple. Anyway https://github.com/Azure/azure-sdk-for-js/pull/10923 will help us standardize those configs anyway.

Reply:
Let's think about this later then. Thank you!